integrated-models/Yeast-GEM/dataOverlay/transcriptomics/protein2.mock.tsv

```diff
@@ -0,0 +1,3 @@
+id	gland 1	gland 2				
+YNR073C	0.81	0.89				
+YOL136C	0.22	0.32				

```

Comment:
```suggestion
id	gland 1	gland 2
YNR073C	0.81	0.89
YOL136C	0.22	0.32
```

Reply:
> > * For each data source, two correct component ids as well as one faulty component id are included.
> 
> `FAULTY_ID` was present only in a couple of files as far as I could see. I don't find these faulty examples necessary, but I am requesting changes on the expectation of consistency with the PR body and between the mock examples. There were a few places with extra tabs that I've found, see the suggestions below.

The `FAULTY_ID` rows are there to represent component ids that do not exist within Metabolic Atlas. They are also used to verify that `neo4j-data-generation`'s processing of data overlay sources are working correctly, as mentioned in [this comment](https://github.com/MetabolicAtlas/private-issues/issues/113#issuecomment-895321963).

Thanks for including the suggested changes. I created these initially not knowing that the extra tabs were there and only discovered them during testing `neo4j-data-generation` and it had [an impact on the implementation](https://github.com/MetabolicAtlas/neo4j-data-generation/pull/20/files#diff-8a397861fc5ef4ff207a6bb2ee81cd030d864ef411604e000b397de4e8b05bd4R100). The [final output](https://github.com/MetabolicAtlas/neo4j-data-generation/pull/20/files#diff-8a397861fc5ef4ff207a6bb2ee81cd030d864ef411604e000b397de4e8b05bd4R111) also ensures that no extra whitespace are found at the beginning or end of rows. I thought it was safer to include this `trim()` logic during processing as there are no official rules about extra whitespace for the raw data files. Having said that, I understand if you still want the mock data files to be cleaner in this repo and I can accept the changes as you requested, just let me know.

Reply:
> The FAULTY_ID rows are there to represent component ids that do not exist within Metabolic Atlas. They are also used to verify that neo4j-data-generation's processing of data overlay sources are working correctly, as mentioned in this comment.

Yeah that makes sense. I was just expect a `FAULTY_ID` in _each_ data source file, as the PR body says, but that's not the case.

> you still want the mock data files to be cleaner in this repo and I can accept the changes as you requested, just let me know.

It's great that you applied the `trim()` for extra robustness. Since these are mock example files, please remove the unneeded tabs (feel free to do all in a single commit, or however you prefer).

Reply:
> Yeah that makes sense. I was just expect a `FAULTY_ID` in _each_ data source file, as the PR body says, but that's not the case.

Ah that's right. I think I changed my mind about it since I wanted to include some files without faulty rows too but I should have updated the description.


> It's great that you applied the `trim()` for extra robustness. Since these are mock example files, please remove the unneeded tabs (feel free to do all in a single commit, or however you prefer).

Sure.

Reply:
Have updated the PR description and removed the extra whitespace in the mock files [here](https://github.com/MetabolicAtlas/data-files/pull/14/commits/49681e596c8c27f15df70de4b6f426cfb3ec98cf).